_events/repeated/community-call-planning.md

```diff
@@ -5,8 +5,9 @@ location: Virtual
 expires: 2025-12-11
 event_date: "fourth Thursday each month 12:00-13:00 Eastern"
 layout: event
+category: community-call
 time:
-  - - start 2022-01-27
+  - - start: 2022-01-27 12:00 EST
```

Comment:
I think probably this was missed because given an RRULE we don't look at this. https://github.com/USRSE/usrse.github.io/blob/6e3708417e1d4f4ee2f7fdaecc3bbc949b389baf/_includes/events/event.js#L20